custom_components/victron_ble/sensor.py

```diff
@@ -2,7 +2,7 @@
 from __future__ import annotations
 
 import logging
-from typing import Any, Dict, Optional, Tuple, Union
+from typing import Any, Dict, Optional, Tuple, Union, Callable
```

Comment:
Very minor pedantry - I'd prefer to see Callable between Any and Dict to preserve the alphabetical ordering. But I wouldn't mind this being committed as-is.